openasip/tools/scripts/install_qemu.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+
+import os
+import shlex
+import subprocess
+import multiprocessing
+import argparse
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description="Clone and build QEMU-OpenASIP for RISC-V instruction emulation"
+    )
+    parser.add_argument(
+        "qemu_path",
+        help="Path to clone/build QEMU-OpenASIP"
+    )
+    return parser.parse_args()
+
+args = parse_args()
+
+QEMU_PATH = args.qemu_path
+qemu_build_dir = os.path.join(QEMU_PATH, 'build')
+qemu_executable = os.path.join(qemu_build_dir, 'qemu-system-riscv32')
+configure_cmd = "./configure --target-list=riscv32-softmmu,riscv64-softmmu"
```

Comment:
This is a bit suspicious since we do not support MMU with the OpenASIP RV designs, do we? What is 'softmmu'?